compiler/x/codegen/CompareAnalyser.cpp

```diff
@@ -143,16 +143,22 @@ void TR_X86CompareAnalyser::integerCompareAnalyser(
       tempMR->decNodeReferenceCounts(cg());
       }
 
-   if (realFirstChild)
-      cg()->recursivelyDecReferenceCount(realFirstChild);
-   else
-      cg()->decReferenceCount(firstChild);
-
-   if (realSecondChild)
-      cg()->recursivelyDecReferenceCount(realSecondChild);
-   else
-      cg()->decReferenceCount(secondChild);
-   }
+   if (realFirstChild) {
```

Comment:
Please use the brace indent style consistent with the rest of the file.  Applies to this `if` and the one below.